src/main/java/Element.java

```diff
@@ -0,0 +1,103 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class Element implements WebElement {
+
+    protected WebDriver driver = BaseDriver.initDriver();
+    protected WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(20));
+    protected Element element;
```

Comment:
Please use WebElement element; without modifications, and create a public metod with name of your class, where you could use in parameters WebElement element and using **this** show it at methods.